src/markdown-pages/docs/upgrading.md

```diff
@@ -34,12 +34,12 @@ Galasa releases 0.35.0 and earlier support the use of Gradle versions 6.8.x, 6.9
 
 To use Gradle version 8.0 or later to create Galasa projects and build and compile Galasa test code, complete the following steps:
 
-1. Edit the `build.gradle` file in the OBR directory of your parent project, setting the plug-in values at the top of the file to version `0.36.0`, as shown in the following example:
+1. Edit the `build.gradle` file in the OBR directory of your parent project, setting the plug-in values at the top of the file to version `0.37.0`, as shown in the following example:
 ```
 plugins {
 ...
-id 'dev.galasa.obr' version '0.36.0'
-id 'dev.galasa.testcatalog' version '0.36.0'
+id 'dev.galasa.obr' version '0.37.0'
+id 'dev.galasa.testcatalog' version '0.37.0'
```

Comment:
0.37.0 versions of the Gradle plugins don't exist since there haven't been any changes to them (e.g. https://mvnrepository.com/artifact/dev.galasa.testcatalog/dev.galasa.testcatalog.gradle.plugin) - the versions can stay at 0.36.0.